extensions/blocks/social-previews/panel.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Social Previews panel component.
+ *
+ * Shows available services and allows opening up the preview modal.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { AVAILABLE_SERVICES } from './constants';
+import { SocialServiceIcon } from '../../shared/icons';
+
+const SocialPreviewsPanel = function SocialPreviewsPanel( { openModal } ) {
+	return (
+		<div className="jetpack-social-previews__panel">
+			<p>
+				{ __(
+					'Preview what this will look like on social networks and Google search.',
```

Comment:
I wonder if we could provide the _type_ of content. So it would read something like

> Preview what this Post will look like on social networks and Google search.

Might be one for a followup?

Reply:
Agreed it would be better but from the i18n experience, I know It's really tough to put the name of the content type into a natural sentence. It works in english but other languages might need a different variant of the word for "post" and that becomes impossible to handle.

What about we ping somebody for the whole copy review once we have all the base pieces together and they can see how things work and interact?

Reply:
Sounds good.